pkg/metaserver/key.go

```diff
@@ -30,30 +30,26 @@ func KeyFuncObj(obj runtime.Object) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	var key string
-	key += "/"
-	gvk := obj.GetObjectKind()
-	if gvk.GroupVersionKind().Empty() {
+
+	objKind := obj.GetObjectKind()
+	gvk := objKind.GroupVersionKind()
+	if gvk.Empty() {
 		return "", fmt.Errorf("could not get group/version/kind information in obj")
 	}
-	group := gvk.GroupVersionKind().Group
-	version := gvk.GroupVersionKind().Version
-	resources := util.UnsafeKindToResource(gvk.GroupVersionKind().Kind)
+	group := gvk.Group
```

Comment:
Could we reduce temp val?

Reply:
`GroupVersionKind()` is not a zero-cost function in most implementations, I don't think it is necessary to use it multiple times.
https://github.com/kubeedge/kubeedge/blob/f30b2021cded5191aa116f8ff51dbc9dc1f677d3/vendor/k8s.io/apimachinery/pkg/runtime/schema/group_version.go#L296-L305